src/qml/components/ChatMessage.qml

```diff
@@ -159,10 +159,29 @@ Control {
 
                     Label {
                         Layout.maximumWidth: column.maxWidth
+                        visible: chatMessage.attachmentStatus == Enums.AttachmentStatus.Loaded
                         text: attachmentDisplaySize
                         color: "white"
                         font.pixelSize: UiHelper.fixFontSz(10)
                     }
+
+                    Label {
+                        Layout.maximumWidth: column.maxWidth
+                        visible: chatMessage.attachmentStatus == Enums.AttachmentStatus.Loading
+                        text: {
+                            if (attachmentBytesLoaded === 0) {
```

Comment:
Much better now, but I meant to fully remove qml calculations. My proposal for the final fix:
- rename AttachmentSizeRole to AttachmentDisplayProgressRole
- move IF inside c++, in qml just leave `text: attachmentDisplayProgress`